realm/realm-jni/src/io_realm_internal_LinkView.cpp

```diff
@@ -176,3 +176,18 @@ JNIEXPORT jboolean JNICALL Java_io_realm_internal_LinkView_nativeIsAttached
     } CATCH_STD()
     return 0;
 }
+
+JNIEXPORT jboolean JNICALL Java_io_realm_internal_LinkView_nativeFind
+  (JNIEnv *env, jobject, jlong nativeLinkViewPtr, jlong rowIndex)
+{
+    TR_ENTER_PTR(nativeLinkViewPtr)
+    try {
+        jlong ndx = LV(nativeLinkViewPtr)->find(rowIndex);
+        if (ndx == to_jlong_or_not_found(realm::not_found)) {
```

Comment:
This looks at bit funny, I am pretty sure that find doesn't return a jlong, so there shouldn't be a need to convert it. But see my comment below, I think this could be refactored to just return the index value. Then we can use this method for other methods like `indexOf`